docs/src/content/docs/index.mdx

```diff
@@ -4,114 +4,137 @@ description: Terrateam documentation
 template: doc
 hero:
   title: Welcome to Terrateam
-  tagline: Terraform and OpenTofu GitOps CI/CD that enables teams to deliver infrastructure faster.
+  tagline: GitOps CI/CD for Terraform and OpenTofu that accelerates infrastructure delivery.
   image:
     file: ../../assets/logo.png
   actions:
     - text: Quickstart guide
       link: /quickstart-guide
       variant: primary
+    - text: Core Concepts
+      link: /getting-started/concepts
+      variant: secondary
 ---
 
 import { Steps } from '@astrojs/starlight/components';
 import { LinkCard, CardGrid, Card } from '@astrojs/starlight/components';
 
 ## What is Terrateam?
```

Comment:
Use bullet points or cards instead of a single text block to improve readability.